src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -90,6 +90,17 @@ public void Configure(OpenIddictServerOptions options)
             if (settings.UserinfoEndpointPath.HasValue)
             {
                 options.UserinfoEndpointUris.Add(new Uri(settings.UserinfoEndpointPath.Value, UriKind.Relative));
+                
```

Comment:
Please remove the extra lines.

Reply:
Removed the extra lines per your request